src/OllamaConfig.php

```diff
@@ -14,6 +14,12 @@ class OllamaConfig
 
     public bool $formatJson = false;
 
+    public ?int $timeout = null;
+
+    public ?int $connect_timeout = null;
```

Comment:
What do you think about using just `$timeout` removing the other 2 properties?

Reply:
I added 2 because ollama supports 2, but actually it could make sense (for cross platform compatibility) to only have one